tedana/decomposition/ica.py

```diff
@@ -50,6 +53,51 @@ def tedica(data, n_components, fixed_seed, maxit=500, maxrestart=10):
         "decompose the dimensionally reduced dataset."
     )
 
+    if ica_method=='robustica':
+        mmix, Iq = r_ica(data, n_components, n_robust_runs, maxit)
+        fixed_seed=-99999
+    elif ica_method=='fastica':
+        mmix, fixed_seed=f_ica(data, n_components, fixed_seed, maxit=500, maxrestart=10)
+        Iq = 0
+    else:
+        LGR.warning("The selected ICA method is invalid!")
+        sys.exit()
+
+
+
+
```

Comment:
Remove excess empty lines

Reply:
All extra spaces and personal notes have been removed.